mindocr/models/backbones/mindcv_models/mobilenet_v3.py

```diff
@@ -188,7 +196,7 @@ def __init__(
         ])
         self.feature_info.append(dict(chs=output_channels, reduction=total_reduction, name=f'features.{len(features) - 1}'))
         self.flatten_sequential = True
-        self.features = nn.SequentialCell(features)
+        self.features = nn.CellList(features)
```

Comment:
The changed architecture seems to be the same as the standard mobilenet under the default value. But here SequentialCell -> CellList, will it affect the model weight loading for the previous defined models like mobilenet v3 small? 
If so, we need to set `auto_mapping` as True when invoke `load_pretrained` in the model definition functions.

Reply:
Backwards compatibility was the highest priority for me. So, no, weights loading will not be affected. We can load MindCV checkpoints (`mobilenet_v3_small_1.0` or `mobilenet_v3_large_1.0`) as well as modified version `mobilenet_v3_large_050_no_scale_se_v2_expand` without any problems.